utils/utils.go

```diff
@@ -125,3 +126,21 @@ func Tar(source string) (io.ReadCloser, error) {
 	logrus.Debugf("creating tarball of %s", source)
 	return archive.Tar(source, archive.Uncompressed)
 }
+
+// RemoveScientificNotationFromFloat returns a float without any
+// scientific notation if the number has any.
+// golang does not handle conversion of float64s that have scientific
+// notation in them and otherwise stinks.  please replace this if you have
+// a better implementation.
+func RemoveScientificNotationFromFloat(x float64) (float64, error) {
```

Comment:
do you have any example that is not handled by `"%.2f"` and it is causing problems?

Reply:
no, this is what was used before and i didnt care to delete it in case there was a problem...